timemachine/fe/interpolate.py

```diff
@@ -148,4 +148,33 @@ def linear_interpolation(src_params, dst_params, lamb):
     """
     Linearly interpolate between src and dst params
     """
-    return (1 - lamb) * np.array(src_params) + lamb * np.array(dst_params)
+    return (1 - lamb) * jnp.asarray(src_params) + lamb * jnp.asarray(dst_params)
+
+
+def log_linear_interpolation(src_params, dst_params, lamb, min_value):
+    """
+    Linear interpolation in log space
+    """
+    return jnp.exp(
+        linear_interpolation(
+            jnp.log(jnp.maximum(src_params, min_value)),
+            jnp.log(jnp.maximum(dst_params, min_value)),
+            lamb,
+        )
+    )
```

Comment:
1. Clarification: if `src_params` and `dst_params` < `min_val`, does this not return `src_params` @ `lamb=0`, `dst_params` @ `lamb=1`?

2. Style opinion: preference for simpler compound statements 
```suggestion
    src = jnp.maximum(src_params, min_value)
    dst = jnp.maximum(dst_params, min_value)
    return jnp.exp(linear_interpolation(jnp.log(src), jnp.log(dst), lamb))
```

Reply:
Ahh, I see -- it's safe if combined with `pad`, which pins to end-state values outside of `[lambda_min, lambda_max]`

Reply:
Yes; also in cases where we don't use `pad` (if there is no ring opening/closing), the end-states are special cases in `interpolate_harmonic_force_constant`

Reply:
Hopefully further addressed potential confusion in 4f91650b4f42380f1df478a098d15865261a4f60.